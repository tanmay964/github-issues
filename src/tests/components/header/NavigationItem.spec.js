import React from 'react';
import renderer from 'react-test-renderer';
import NavigationItem from '../../../components/header/NavigationItem';
import { OWNER, REPO, WEB_URL } from '../../../api/constants';

it('renders correctly', () => {
  const navigationValues = [
    {
      name: 'Code', selected: false, url: `${WEB_URL}/${OWNER}/${REPO}`,
    },
    { name: 'Issues', value: 253, selected: true },
    {
      name: 'Pull Requests', value: 72, selected: false, url: `${WEB_URL}/${OWNER}/${REPO}/pulls`,
    },
    {
      name: 'Projects', value: 2, selected: false, url: `${WEB_URL}/${OWNER}/${REPO}/projects`,
    },
    {
      name: 'Insights', selected: false, url: `${WEB_URL}/${OWNER}/${REPO}/insights`,
    },
  ];
  const tree = renderer
    .create(<NavigationItem {...navigationValues} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
