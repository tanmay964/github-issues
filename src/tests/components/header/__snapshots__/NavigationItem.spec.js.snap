// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bwzfXH fOTHXW"
>
  <a
    className="sc-bxivhb dHCaOF"
  />
  <span />
</div>
`;
